plugin/pprof/setup.go

```diff
@@ -47,10 +47,8 @@ func setup(c *caddy.Controller) error {
 		}
 	}
 
-	pprofOnce.Do(func() {
-		c.OnStartup(h.Startup)
-		c.OnShutdown(h.Shutdown)
-	})
+	c.OnStartup(h.Startup)
+	c.OnRestart(h.Shutdown)
```

Comment:
Hum.. that will work for the reload.
However, if the pprof setup is called several time for each reload, only the first setup will succeed, and following will fail the chain of startup event.

It may be a limitation we do not care ... just need to say it in the README.

exemples:

```
mydomain.com, other.com {
     pprof ...
}

```
or

```
. {
   pprof ...
}
mydomain.com {
   pprof ...
}

```

Reply:
yes, good question; does this still work for this scenario?